test/unit/bootstrap.js

```diff
@@ -23,9 +23,11 @@ global.fetch = fetch;
 global.fetch.Promise = Promise;
 
 // this could be replaced by jsdom.Navigator in https://github.com/iTowns/itowns/pull/1412
-global.navigator = {
-    userAgent: 'firefox',
-};
+if (!global.navigator) {
```

Comment:
Could you add a notice that such code targets `node < 21.0.0`? This could ease its removal once our minimum node targets will be `>= 21.0.0`. Something like:
```suggestion
// Partial support for global.navigator was added in node 21.0.0
if (!global.navigator) {
```

Reply:
Done